spec/problem2_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'problem2'
+describe 'problem2' do
+  it 'returns the two largest values from sorted array' do
+    expect(problem2([1,2,3,4,5])).to match_array [4,5]
+  end
+
+  it 'returns the two largest values from reverse sorted array' do
```

Comment:
I just now noticed that you don't have a test for negative numbers, it would have been more relevant that this one.